src/js/base/module/Editor.js

```diff
@@ -442,7 +447,20 @@ export default class Editor {
    */
   createRange() {
     this.focus();
-    return range.create(this.editable);
+    // return range.create(this.editable);
```

Comment:
I saw that there are many `range.create` calling at internal and external modules. Is there a chance to use `range.create` from outside of `Editor.js` module?

Reply:
It would be better to remove comment.

Reply:
ok, i will remove comment.